front/src/common/Pathfinding/Pathfinding.tsx

```diff
@@ -249,7 +253,75 @@ function Pathfinding({ mustUpdate = true, zoomToFeature }: PathfindingProps) {
     rollingStockID,
   };
   const [pathfindingState, pathfindingDispatch] = useReducer(reducer, initializerArgs, init);
+  const [isInfraLoaded, setIsInfraLoaded] = useState(false);
+  const [reloadCount, setReloadCount] = useState(1);
+  const [isInfraError, setIsInfraError] = useState(false);
   const [postPathfinding] = osrdMiddlewareApi.usePostPathfindingMutation();
+  const { data: infra } = osrdEditoastApi.useGetInfraByIdQuery(
+    { id: infraID as number },
+    {
+      refetchOnMountOrArgChange: true,
```

Comment:
Is this needed ?
```suggestion
      refetchOnMountOrArgChange: true,
```

Reply:
I think so yes we will discuss it together :)